uint8 masterID
string stateReq
---
int16 stateRes
